### PR TITLE
Overhaul building-archlinux-template.md

### PR DESCRIPTION
**Pull request by ghost-420**

### CODE REVIEW

File: docs/building/building-archlinux-template.md [hunk 1]
```diff
@@ -1,152 +1,243 @@
+# These instructions are for Qubes 4.0.? and 4.1.?
 
-Archlinux template building instructions
-===========================================
```
Comment (awokd) [inline docs/building/building-archlinux-template.md:3]: Could you please add the title back at the top, and move the note about Qubes versions below it?

**Comment by awokd** [comment]
Overall looks good, have you tested the procedure from start to end?

**Comment by awokd** [comment]
@ghost-420 ?

**Comment by ghost-420** [comment]
> Overall looks good, have you tested the procedure from start to end?

Yes, it will fail on Qubes OS 4.1; old instructions also fail.

**Comment by awokd** [comment]
Thank you.

**Pull request merged**